.github/workflows/test.yaml

```
@@ -130,7 +130,7 @@ jobs:
     secrets: inherit
     uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-24.06
     with:
-      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
+      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | min(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
```

```suggestion
      matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(min_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
```

I believe that results in this matrix:
```

  "include": [
    {
      "ARCH": "amd64",
      "PY_VER": "3.9",
      "CUDA_VER": "11.8.0",
      "LINUX_VER": "ubuntu20.04",
      "gpu": "v100",
      "driver": "latest"
    },
    {
      "ARCH": "amd64",
      "PY_VER": "3.9",
      "CUDA_VER": "12.2.2",
      "LINUX_VER": "ubuntu22.04",
      "gpu": "v100",
      "driver": "latest"
    }
  ]
}
```